### ISSUE TITLE
Listbox: unable to change listbox selection with Chrome & Talkback on Android

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/mindpatterns#131]
Steps: 

1. Visit https://opensource.ebay.com/mindpatterns/input/listbox/index.html
2. Enable Talkback (hold both volume keys)
3. Swipe to listbox
4. Double tap to activate listbox
5. Swipe to change highlighted option
6. Double tap to select the highlighted option

Steps 3 and 4 are failing in MIND pattern example

<hr>

All steps are working in the APG main documentation page:

https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-scrollable/

However, these steps are NOT working in the APG codepen view. Odd!

**Comment by ianmcburnie** [comment]
Possible issue with certain versions of chrome/android/talkback: https://issues.chromium.org/issues/40260928
